### ISSUE TITLE
Jasp freezes and hardrive goes to 90-100% when I start the program. analysis function grayed out

### ISSUE THREAD

**Issue by thesaw123** [jasp-stats/jasp-issues#403]
* JASP version:
* OS name and version:
* Analysis:
* Bug description:
* Add JASP files as a zip:
* Screenshot: 
* Expected behaviour:
* Steps to reproduce:

**Comment by thesaw123** [comment]
![Screenshot 2019-06-26 12 35 42](https://user-images.githubusercontent.com/52257824/60209468-9a2bbd80-980f-11e9-8515-cc3808f0f713.png)
I have installed version 0.10 of Jasp.  Initially it ran fine, however, after running a few analysis it began to act buggy.  It would either freeze up or the analysis would be blank.  Then it began crashing as soon as I tried to open a file, and now it will not even function.  All of the analysis icons are grayed out except for summary statistics.  I also noticed that when I start the program my disk function shoots up to nearly 100%.  I am running HP w 12Gb of ram on a windows10 system w plenty of harddrive space.  Any suggestions as to how to resolve this would be appreciated.
